dynawo/sources/Common/DYNTrace.h

```diff
@@ -230,12 +232,12 @@ class Trace {
     }
 
     /**
-     * @brief Set the append attribute
+     * @brief Set the openmode of the log file
      *
-     * @param append determines if the log must appended to existing file
+     * @param openMode openMode of the log file
      */
-    void setAppend(bool append) {
-      append_ = append;
+    void setOpenMode(std::ios_base::openmode openMode) {
```

Comment:
This might break dynawo-algorithmes and/or DFL

Reply:
(and I feel it is out of the scope of this ticket)

Reply:
This attribute seems to be never actually used. I removed it to make the code more simple and more reusable with functions.